.github/workflows/benchmark_asterinas.yml

```diff
@@ -59,6 +59,7 @@ jobs:
           - lmbench/tcp_loopback_select_lat
           - lmbench/tcp_loopback_http_bw
           - lmbench/udp_loopback_lat
+          - iperf3/tcp_bw
```

Comment:
The `tcp_bw` can be renamed to `tcp_bw_host_guest` or `tcp_bw_virtio` since we will add the bandwidth of loopback soon.